src/main/java/Bob.java

```diff
@@ -0,0 +1,157 @@
+            completeTask(inputStringArr[1]);
+            break;
+        case TODO:
+            addTodo(inputString);
+            break;
+        case DEADLINE:
+            addDeadline(inputString);
+            break;
+        case EVENT:
+            addEvent(inputString);
+            break;
+        case BYE:
+            isScanning = false;
+            break;
+        default: //regular task
+            addTask(inputString);
+        }
+        return isScanning;
+    }
+
+    private static void addTask(String inputString) {
+        taskList.addToList(new Task(inputString));
+    }
+
+    private static void completeTask(String s) {
+        taskList.completeTask(Integer.parseInt(s));
+    }
+
+    private static void printList() {
+        taskList.printList();
+    }
+
+    /**
+     * Takes a string and returns a enum Command
+     * Treats an unknown command as regular task
+     */
+    private static Command getCommandType(String commandString) {
+        Command command;
+        switch (commandString) {
+        case "list":
+            command = Command.LIST;
+            break;
+        case "done":
+            command = Command.DONE;
+            break;
+        case "todo":
+            command = Command.TODO;
+            break;
+        case "deadline":
+            command = Command.DEADLINE;
+            break;
+        case "event":
+            command = Command.EVENT;
+            break;
+        case "bye":
+            command = Command.BYE;
+            break;
+        default:
+            command =  Command.TASK;
+        }
+        return command;
+    }
+
+    private static void addTodo(String command) {
+        String label = command.substring("todo".length() + 1);
+        taskList.addToList(new Todo(label));
+    }
+
+    /**
+     * Phrases command into label and startTime parts
+     * Followed by creating event object
+     */
+    private static void addEvent(String command) {
+        final String startTimeMarker = "/at";
+        String commandType = Command.EVENT.name().toLowerCase();
+
+        String[] commandArray = command.split(startTimeMarker);
+        String label = getLabel(commandArray[0],commandType);
+        String startTime = commandArray[1].trim();
+
+        taskList.addToList(new Event(label, startTime));
+    }
+
+    /**
+     * Phrases command into label and dueTime parts
+     * Followed by creating Deadline object
+     */
+    private static void addDeadline(String command) {
+        final String dueTimeMarker = "/by";
+        String commandType = Command.DEADLINE.name().toLowerCase();
+
+        String[] commandArray = command.split(dueTimeMarker);
+        String label = getLabel(commandArray[0],commandType);
+        String dueTime = commandArray[1].trim();
+
+        taskList.addToList(new Deadline(label, dueTime));
+    }
+
+    private static String getLabel(String string, String commandType) {
+        return string.replaceFirst(commandType,"").trim();
```

Comment:
It's would look better if can add a space between the double quotation mark. eg." "

Reply:
Well it's intentional to replace the command with an empty string at the moment